packages/validator/src/services/block.ts

```diff
@@ -110,6 +110,7 @@ export class BlockProposingService {
         builderSelection,
       }).catch((e: Error) => {
         this.metrics?.blockProposingErrors.inc({error: "produce"});
+        this.metrics?.missedBlocks.inc();
```

Comment:
There's already a metric in the previous line, why this duplicate metric necessary?

Reply:
"missed block" is a fuzzy term that usually refers to a block that did not make it on-chain. Block production error is one of the reasons, but could be more which you cannot track from the validator client

Reply:
Correct, just to clarify my understanding is a _missed block_ is one where a validator node fails to validate a block that it was selected to validate and yes this is just one instance of it (hence there is a separate metric i.e. `missedBlocks`)

Reply:
You should not create duplicate metrics, since in Grafana you can easily aggregate then with dedicated queries